### PR TITLE
System > Common Good Parachains

### PR DESCRIPTION
**Pull request by joepetrowski**

### CODE REVIEW

File: docs/build/build-hrmp-channels.md [hunk 1]
```diff
@@ -28,23 +28,22 @@ In order to do this, the calls to be dispatched from both chains are:
 Each parachain is to use its own governance process to execute this. The call will be executed on
 the Relay Chain.
 
-## Opening HRMP channel to Statemint
+## Opening HRMP Channels with System Parachains
 
-In order to provide complete contextual information and transparency to the community, which is the
-ultimate decision maker of the proposals, we encourage following the process implemented on Kusama.
+Opening an HRMP channel with a system parachain requires a referendum. Like all other governance
+proposals, proposers should follow best practices like opening a discussion on Polkassembly or
```
Comment (profradha) [inline docs/build/build-hrmp-channels.md]:
```suggestion
proposals, proposers should follow best practices like opening a discussion on [Polkassembly](https://polkadot.polkassembly.io/) or
```

File: docs/build/build-hrmp-channels.md [hunk 2]
```diff
@@ -28,23 +28,22 @@ In order to do this, the calls to be dispatched from both chains are:
...
+Subsquare and then submitting the proposal on-chain.
```
Comment (profradha) [inline docs/build/build-hrmp-channels.md]:
```suggestion
[Subsquare](https://polkadot.subsquare.io/) and then submitting the proposal on-chain.
```

File: docs/learn/learn-parachains.md [hunk 3]
```diff
@@ -169,16 +169,15 @@ maintain the stronger property of single block composability.
...
 
-["Common Good" parachains](#common-good-parachains) are allocated by
+["System" parachains](#system-parachains) are allocated by
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}'s on-chain
-[governance](learn-governance.md) system, and are deemed as a "common good" for the network, such as
-bridges to other networks or chains. They are usually considered system-level chains or public
-utility chains. These typically do not have an economic model and help remove transactions from the
-Relay Chain, allowing for more efficient parachain processing.
+[governance](learn-governance.md) system, and are deemed as part of the network's protocol, such as
+bridges to other networks or chains. These typically do not have an economic model and help remove\
```
Comment (profradha) [inline docs/learn/learn-parachains.md]:
```suggestion
bridges to other networks or chains. These typically do not have an economic model and help remove 

```

**Review by DrW3RK — APPROVED** [review]
Looks good to me! Implemented redirect for the common good parachain doc to the the newly created systems parachain doc, which ensures the wiki links on third party websites are not broken. 

Also, IMO Encointer seems to not fit the template definition of a system parachain.

**Pull request merged**